File: c/4-functions/practice/fp-1.c
#include <stdio.h>
int main()
{
    int a,count =0;
    scanf("%d",&a);
    if(a<=0)
    {
        printf("Invalid Input");
    }
    else
    {
        while(a>0)
        {
            if(a>=3)
            {
              count++;
              a=a-3;
            }
            else if(a>=2)
            {
                count++;
                a=a-2;
            }
            else
            {
                count++;
                a=a-1;
            }
        }
        printf("%d",count);
    }
    
}

File: python/5-functions/practice/fp-1.py
def calc(n):
    sum=0
    product=1
    for i in n:
        sum+=i
        product*=i
    return sum,product
try:
    n=list(map(int,input().split()))
    s,p=calc(n)
    print(f"Sum of elements: {s}")
    print(f"Product of elements: {p}")
except:
    print("Invalid input")